mod_auth/controllers.py

```diff
@@ -117,6 +117,26 @@ def send_reset_email(usr):
               'error-message')
 
 
+@mod_auth.route('/github_token_validity', methods=['GET', 'POST'])
+def github_token_validity(token):
+    """
+    Check token validity
+    Set token to null if not valid
```

Comment:
This def does not change the token of a user.

Argument type & description are missing, just as the return values.